paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc

```diff
@@ -23,15 +24,34 @@ class Graph;
 
 void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
   VLOG(3) << "Marks operators which are to be quantized.";
+  std::unordered_set<std::string> supported_op_types =
+      std::unordered_set<std::string>(
+          {"concat", "conv2d", "depthwise_conv2d", "elementwise_add", "fc",
+           "matmul", "nearest_interp", "nearest_interp_v2", "pool2d",
+           "prior_box", "reshape2", "transpose2", "fusion_gru", "fusion_lstm",
```

Comment:
Just for my curiosity, what is `prior_box`? Do we support that operation?

Reply:
I have found it already, I think that it is the only op that doesn't end with "mkldnn_op.cc"

Reply:
Yes, you are right, we are using a native version here which supports all data types including int8.